### PR TITLE
Fix this.params.cache to allow disabling cache & add test for it

### PR DESCRIPTION
**Pull request by apapirovski**
This should address https://github.com/schemaio/schema-node-client/issues/4

Let me know if any changes are needed! Thanks.

### CODE REVIEW

File: lib/client.js [hunk 1]
```diff
@@ -66,7 +66,7 @@ Client.prototype.init = function(clientId, clientKey, options) {
     route: options.route,
     timeout: options.timeout,
     routeClientId: options.route && options.route.client,
-    cache: options.cache || true,
+    cache: typeof options.cache !== 'undefined' ? options.cache : true
```
Comment (apapirovski) [inline lib/client.js:69]: I didn't include a trailing comma since it doesn't match your current lint rules. Should probably be taken care of in a separate PR if you want to start including those.

Comment (ericingram) [reply lib/client.js:69]: Good catch!

**Comment by ericingram** [comment]
Looks good to me 👍
